Lib/test/test_compile.py

```diff
@@ -947,6 +947,17 @@ def return_genexp():
                       for (_, _, line) in genexp_code.co_lines() ]
         self.assertEqual(genexp_lines, code_lines)
 
+    def test_line_number_implicit_return_after_async_for(self):
+
+        async def test(aseq):
+            async for i in aseq:
+               body
```

Comment:
Travis CI checks are failing ``make patchcheck``
```suggestion
            async for i in aseq:
                body
```